index.html

```diff
@@ -149,8 +149,6 @@ <h3 id="modal-message">sign in</h3>
     <script src="assets/javascript/logic.js"></script>
     <script src="assets/javascript/events.js"></script>
   </body>
-
-  <footer>
                            
                        <li>
```

Comment:
This whole section needs to be deleted. All of the <li> tags and links should be deleted. Everything needs to go.